pilot/helpers/agents/SpecWriter.py

```diff
@@ -31,20 +36,42 @@ def analyze_project(self, initial_prompt):
             if not llm_response:
                 continue
 
+            num_questions += 1
             llm_response = llm_response.strip()
             if len(llm_response) > 500:
+                print('continue', type='button')
                 user_response = ask_user(
                     self.project,
-                    "Can we proceed with this project description? If so, just press ENTER. Otherwise, please provide a comment, addition or change you want in this specification.",
+                    "Can we proceed with this project description? If so, just press ENTER. Otherwise, please tell me what's missing or what you'd like to add.",
                     hint="Does this sound good, and does it capture all the information about your project?",
                     require_some_input=False
                 )
                 if user_response:
                     user_response = user_response.strip()
-                if user_response.lower() in ["", "yes", "ok", "y", "continue"]:
+                if user_response.lower() in AFFIRMATIVE_ANSWERS + ['continue']:
                     break
             else:
+                print('skip questions', type='button')
                 user_response = ask_user(self.project, llm_response)
+                if user_response == 'skip questions':
```

Comment:
shouldn't this be `if user_response.lower() == 'skip questions':` ?

Reply:
Not needed, because the extension uses the same case that the buttons are specified (they are only capsed for the button label):

![Screenshot from 2024-02-17 17-23-47](https://github.com/Pythagora-io/gpt-pilot/assets/3362/1ff0476b-59a6-42b2-905a-55b682c9b146)

Added anyway to avoid potential future breakage if extension changes.